### PR TITLE
BIBSELV-256: Updated restart script to non-docker hosting

### PR DESCRIPTION
**Pull request by turegjorup**
https://jira.itkdev.dk/browse/BIBSELV-256

* Updated `restart.sh` to work on a non-docker server
* Updated shell scripts to set correct working directory regardless of call context
* Updated Readme

### CODE REVIEW

File: engine/scripts/audit.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+cd "$(dirname "$0")"
```
Comment (cableman) [inline engine/scripts/audit.sh:2]: I don't understand this "$0" it that the first input parameter given to the script? So change dir, why is that needed?

File: restart.sh [hunk 2]
```diff
@@ -1,21 +1,21 @@
 #!/bin/sh
+cd "$(dirname "$0")"
 
-# Pull and up containers
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml pull
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml up --force-recreate --detach --remove-orphans
```
Comment (cableman) [inline restart.sh:5]: So you have made a `restart-dev.sh` script or?

**Review by cableman — CHANGES_REQUESTED** [review]

File: engine/scripts/audit.sh (hunk 1 above)
Comment (turegjorup) [reply engine/scripts/audit.sh:2]: Without this the script runs in the context of the `dir` it was called from. This ensures it runs in the context of the `dir` it is located in.

From https://fedingo.com/how-to-set-current-working-directory-to-directory-of-shell-script/

File: restart.sh (hunk 2 above)
Comment (turegjorup) [reply restart.sh:5]: Don't see why we should need a `dev` script. Should be handled by our docker setup?

**Review by cableman — APPROVED** [review]
